gp-populate-anything/gppa-populate-child-entries.php

```diff
@@ -66,7 +66,20 @@ public function populate_parent_entry_id( $value, $field ) {
 		} else {
 			$session = new GPNF_Session( $field->formId );
 			$cookie  = $session->get_cookie();
-			$value   = rgar( $cookie, 'hash', '' );
+
+			// try to find cookie, if available.
+			if ( ! $cookie ) {
```

Comment:
So a cookie exists and but it's not retrieved by the session? Could you explain more about this scenario and why the cookie is not being found?

Reply:
The standard cookie `gpnf_form_session` wasn't there, which I thought was to do with some logic on the snippet. But there was another one with the a suffix of `gfpreview` and form Id (forgot the exact format) available. The proposal here is to look out for that if the standard isn't available.

Reply:
I think we'll want to identify why the expected cookie isn't there and why the expected cookie is there.